source/_integrations/co2signal.markdown

```diff
@@ -63,7 +68,7 @@ When configuring the location based on a country code, you will be prompted to e
 
 {% configuration_basic %}
 Country code:
-  description: The two-letter ISO 3166-1 alpha-2 country code for your home location (e.g., "US" for the United States, "DE" for Germany).
+  description: The country code for your home location. For most countries, this is the two-letter ISO 3166-1 alpha-2 country code (for example, `DE` for Germany, `GB` for Great Britain). However, for the United States, you need to specify a zone identifier that includes the specific electricity grid region, such as `US-CENT-SWPP` for the Southwest Power Pool, `US-CAR-DUK` for Duke Energy Carolinas, or `US-CAR-CPLE` for Duke Energy Progress East. You can find zone identifiers in the [Electricity Maps app](https://app.electricitymaps.com) or using the [Zones API reference](https://portal.electricitymaps.com/developer-hub/api/reference#zones).
```

Comment:
This is not limited to the US, we recommend using the most granular zone.

For example Denmark has 3, Norway has 5 and Sweden has 4.